### PR TITLE
Cumulus 296 logging and schemaLocation

### PR DESCRIPTION
**Pull request by kkelly51**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -55,23 +55,27 @@ function callCumulusMessageAdapter(command, input) {
  * If a Cumulus Remote Message is passed, fetch it and return a full Cumulus Message
  *
  * @param {Object} cumulusMessage - either a full Cumulus Message or a Cumulus Remote Message
+ * @param {String} schemaLocation - location of schema file, can be null
```
Comment (laurenfrederick) [inline index.js]: The schema location should be an Object that includes Strings for the input, output, and config schemas.

File: index.js [hunk 2]
```diff
@@ -138,7 +146,7 @@ function runCumulusTask(taskFunction, cumulusMessage, context, callback) {
     );
   }
   else {
-    const promisedRemoteEvent = loadRemoteEvent(cumulusMessage);
+    const promisedRemoteEvent = loadRemoteEvent(cumulusMessage, schemaLocation);
```
Comment (laurenfrederick) [inline index.js]: I am confused about where `schemaLocation` comes from? I think it should be an optional param to `runCumulusTask`.

**Comment by kkelly51** [comment]
Opinion: 
I think it may be easier for task writers to pass in 1 JSON object with any schema locations rather than up to 3 different strings of the location names. Let me know if you disagree and I can change this.

File: README.md [hunk 3]
```diff
@@ -54,14 +54,17 @@ must be communicated to those using the task.
...
-`runCumulusTask`. It takes four parameters:
+`runCumulusTask`. It takes five parameters:
 
   * `taskFunction` - the function containing your business logic (as described
     above)
   * `cumulusMessage` - the event passed by Lambda, and should be a Cumulus
     Message
   * `context` - the Lambda context
   * `callback` - the callback passed by Lambda
+  * `schemas` - the locations of the task schemas
```
Comment (laurenfrederick) [inline README.md]: If you're expecting the schemas in a certain format, specify the format.

File: package-lock.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cumulus/cumulus-message-adapter-js",
-  "version": "0.0.1",
+  "version": "0.0.1-beta.2",
```
Comment (laurenfrederick) [inline package-lock.json]: I think beta.3 is the current version and this should be updated in package.json, not package-lock
https://www.npmjs.com/package/@cumulus/cumulus-message-adapter-js

Comment (kkelly51) [reply package-lock.json]: merge in master to get the latest version

**Pull request merged**